pkg/client/cli/license.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+)
+
+type licenseInfo struct {
+	outputFile string
+}
+
+func LicenseCommand() *cobra.Command {
+	li := &licenseInfo{}
+	cmd := &cobra.Command{
+		Use:  "license",
+		Args: cobra.MinimumNArgs(1),
+
+		Short: "Get License from Ambassador Cloud",
+		Long:  "Get License from Ambasssador Cloud",
+		RunE:  li.getCloudLicense,
+	}
+	flags := cmd.Flags()
+
+	flags.StringVarP(&li.outputFile, "output-file", "f", "/tmp/license", "The file where you want the license secret to be output to")
+	return cmd
+}
+
+func (li *licenseInfo) getCloudLicense(cmd *cobra.Command, args []string) error {
```

Comment:
FWIW, it's my preference to have the separate `RunE` implementation  take arguments like it would if it were a regular Go function, and translate cobra to that in-line; something like:

```go
func LicenseCommand() *cobra.Command {
	var flags struct {
		outputFile string
	}
	cmd := &cobra.Command{
		Use:  "license [flags] <id>",
		Args: cobra.imumNArgs(1),

		Short: "Get License from Ambassador Cloud",
		Long:  "Get License from Ambassador Cloud",
		RunE: func(cmd *cobra.Command, args []string) error {
			return getCloudLicense(cmd.Context(), cmd.OutOrStdout(),
				args[0], flags.outputFile)
		},
	}
	cmd.Flags().StringVarP(&flags.outputFile, "output-file", "f", "/tmp/license", "The file where you want the license secret to be output to")
	return cmd
}
```